Project-3/Ashutosh/README.md

```diff
@@ -0,0 +1,53 @@
+# Weather App
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!